plugins/eddn.py

```diff
@@ -213,7 +219,21 @@ def send(self, cmdr: str, msg: Mapping[str, Any]) -> None:
             ('message', msg['message']),
         ])
 
-        r = self.session.post(self.eddn_url, data=json.dumps(to_send), timeout=self.TIMEOUT)
+        # About the smallest request is going to be (newlines added for brevity):
+        # {"$schemaRef":"https://eddn.edcd.io/schemas/shipyard/2","header":{"softwareName":"E:D Market
+        # Connector Windows","softwareVersion":"5.3.0-beta4extra","uploaderID":"abcdefghijklm"},"messa
+        # ge":{"systemName":"delphi","stationName":"The Oracle","marketID":128782803,"timestamp":"xxxx
+        # -xx-xxTxx:xx:xxZ","ships":[]}}
```

Comment:
Change this to the actually-validates-currently:

```json
{"$schemaRef":"https://eddn.edcd.io/schemas/commodity/3","header":{"softwareName":"E:D Market Connector Windows","softwareVersion":"5.3.0-beta4extra","uploaderID":"abcdefghijklm"},"message":{"systemName":"delphi","stationName":"The Oracle","marketId":128782803,"timestamp":"2022-01-26T12:00:00Z","commodities":[]}}
```

which is 315 bytes (includes the terminating `\n`) and in a test compressed to 244 bytes, so still a saving, if not the 90% saving seen on large messages.

Please do cite the above paragraph too.